src/test/java/org/symphonyoss/integration/jira/webhook/parser/v1/JiraParserUtilsTest.java

```diff
@@ -1,80 +0,0 @@
-/**
- * Copyright 2016-2017 Symphony Integrations - Symphony LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.symphonyoss.integration.jira.webhook.parser.v1;
-
-import org.junit.Assert;
-import org.junit.Test;
-
-/**
- * Unit tests for {@link JiraParserUtils}.
- *
- * Created by mquilzini on 18/05/16.
- */
-public class JiraParserUtilsTest {
```

Comment:
I believe this test shouldn't be removed because the JiraParserUtils is being just moved.